packages/core/admin/admin/src/content-manager/components/DynamicZone/components/tests/ComponentCard.test.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+
+import GlobalStyle from '../../../../../components/GlobalStyle';
+
+import ComponentCard from '../ComponentCard';
+
+describe('ComponentCard', () => {
+  const setup = (props) =>
+    render(
+      <ThemeProvider theme={lightTheme}>
+        <ComponentCard {...props}>test</ComponentCard>
+        <GlobalStyle />
+      </ThemeProvider>
+    );
+
+  it('should render children by default', () => {
+    const { getByTestId, getByText } = setup();
+    expect(getByText('test')).toBeInTheDocument();
+    expect(getByTestId('component-card-icon')).toMatchInlineSnapshot(`
```

Comment:
I think we were trying to remove inline snapshots, should we give them their own file for snapshots on those tests? no strong opinion!

Reply:
This feels subjective, what's the general vibe for it? I feel snapshots in another file are very easily forgotten / ignored at but i'm happy to do that. I think this is something that we need to add to an FE guide.

Reply:
My vibe is: no inline snapshots. 💥